gremlin-go/pom.xml

```diff
@@ -97,7 +97,7 @@ limitations under the License.
                                     <skip>${skipTests}</skip>
                                     <environmentVariables>
                                         <GREMLIN_SERVER>${project.version}</GREMLIN_SERVER>
-                                        <ABS_PROJECT_HOME>${project.basedir}</ABS_PROJECT_HOME>
+                                        <ABS_PROJECT_HOME>${project.basedir}/../</ABS_PROJECT_HOME>
```

Comment:
why is this change necessary?

Reply:
Because `project.basedir` actually resolves to `tinkerpop/gremlin-python`, but we need `tinkerpop`

Reply:
it's `gremlin-go`

Reply:
Yes, it should be changed for all glvs actually. The reason it didn't appear to affect `gremlin-go` is b/c this is needed to set up Kerberos, and no other GLV uses Kerberos than python so when it errored on GHA it wasn't noticed.